tests/tests/test-filter-api.ts

```diff
@@ -151,64 +160,68 @@ describeWithMoonbeam("Moonbeam RPC (EthFilterApi)", `simple-specs.json`, (contex
         topics: ["0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef"],
       },
     ]);
-    expect(create_filter.result).to.be.eq("0x1");
+    expect(createFilter.result).to.be.eq(context.web3.utils.numberToHex(1));
   });
 
-  step("should increment filter ID", async function () {
-    let create_filter = await customRequest(context.web3, "eth_newFilter", [
+  it("should increment filter ID", async function () {
+    let createFilter = await customRequest(context.web3, "eth_newFilter", [
       {
         fromBlock: "0x1",
         toBlock: "0x2",
         address: "0xC2Bf5F29a4384b1aB0C063e1c666f02121B6084a",
         topics: ["0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef"],
       },
     ]);
-    expect(create_filter.result).to.be.eq("0x2");
+    expect(createFilter.result).to.be.eq(context.web3.utils.numberToHex(1));
   });
 
-  step("should create a Block filter and return the ID", async function () {
-    let create_filter = await customRequest(context.web3, "eth_newBlockFilter", []);
-    expect(create_filter.result).to.be.eq("0x3");
+  it("should create a Block filter and return the ID", async function () {
+    let createFilter = await customRequest(context.web3, "eth_newBlockFilter", []);
+    expect(createFilter.result).to.be.eq(context.web3.utils.numberToHex(1));
   });
 
-  step(
-    "should return unsupported error for Pending Transaction filter creation",
-    async function () {
-      let r = await customRequest(context.web3, "eth_newPendingTransactionFilter", []);
-      expect(r.error).to.include({
-        message: "Method not available.",
-      });
-    }
-  );
-
-  step("should return responses for Block filter polling.", async function () {
-    let block = await context.web3.eth.getBlock(0);
-    let poll = await customRequest(context.web3, "eth_getFilterChanges", ["0x3"]);
+  it("should return unsupported error for Pending Transaction filter creation", async function () {
+    let r = await customRequest(context.web3, "eth_newPendingTransactionFilter", []);
```

Comment:
I'm not a big fan of single letter varaibles, except when it is in a single line or an iterator.